src/Blumchen.DependencyInjection/Blumchen.DependencyInjection.csproj

```diff
@@ -0,0 +1,49 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+      <VersionPrefix>0.1.1</VersionPrefix>
+      <TargetFramework>net8.0</TargetFramework>
+      <GenerateAssemblyTitleAttribute>true</GenerateAssemblyTitleAttribute>
+      <GenerateAssemblyDescriptionAttribute>true</GenerateAssemblyDescriptionAttribute>
+      <GenerateAssemblyProductAttribute>true</GenerateAssemblyProductAttribute>
+      <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+      <GenerateAssemblyVersionAttribute>true</GenerateAssemblyVersionAttribute>
+      <GenerateAssemblyFileVersionAttribute>true</GenerateAssemblyFileVersionAttribute>
+      <GenerateAssemblyInformationalVersionAttribute>true</GenerateAssemblyInformationalVersionAttribute>
+      <GenerateDocumentationFile>true</GenerateDocumentationFile>
+      <LangVersion>12.0</LangVersion>
+      <Authors>Oskar Dudycz</Authors>
+      <!--        <PackageIconUrl>https://github.com/event-driven-io/Blumchen/content/images/emblem.png</PackageIconUrl>-->
+      <PackageProjectUrl>https://github.com/event-driven-io/Blumchen</PackageProjectUrl>
+      <PackageLicenseExpression>MIT</PackageLicenseExpression>
+      <RepositoryUrl>https://github.com/event-driven-io/Blumchen.git</RepositoryUrl>
+      <PublishRepositoryUrl>true</PublishRepositoryUrl>
+      <Product>Blumchen</Product>
+      <GenerateDocumentationFile>true</GenerateDocumentationFile>
+      <EmbedUntrackedSources>true</EmbedUntrackedSources>
+      <IncludeSymbols>true</IncludeSymbols>
+      <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+      <RootNamespace>Blumchen</RootNamespace>
+      <PublishAot>true</PublishAot>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+        <NoWarn>1591</NoWarn>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+        <NoWarn>1591</NoWarn>
+    </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
+    <PackageReference Include="Polly" Version="8.4.1" />
```

Comment:
**QUESTION:** Have you thought about just using Microsoft.Resilience package? https://learn.microsoft.com/en-us/dotnet/core/resilience/?tabs=dotnet-cli

Reply:
It seems nothing but an extension method to Polly with some facility to OTEL.
Ww may return on that when approaching telemetry

Reply:
That's fair then :+1: